tests/functional/cli/test_cli_artifacts.py

```diff
@@ -24,12 +24,18 @@
 
 @pytest.fixture(scope="module")
 def job_with_artifacts(gitlab_runner, project):
+    start_time = time.time()
+
     project.files.create(data)
 
     jobs = None
     while not jobs:
         time.sleep(0.5)
         jobs = project.jobs.list(scope="success")
+        if time.time() - start_time > 60:
+            for job in jobs:
+                job.pprint()
+            assert None, "Fixture 'job_with_artifact' failed"
```

Comment:
Are these leftover from debugging?

Reply:
No, those are in case it fails we can get some debug info.